src/woo_publications/metadata/api/tests/test_information_category_api_endpoints.py

```diff
@@ -0,0 +1,152 @@
+from django.urls import reverse
+
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from woo_publications.metadata.constants import InformationCategoryOrigins
+from woo_publications.metadata.tests.factories import InformationCategoryFactory
+
+
+class InformationCategoryTests(APITestCase):
+    @classmethod
+    def setUpTestData(self):
+        super().setUpTestData()
+        self.information_category_value_list = InformationCategoryFactory.create(
+            identifier="https://www.example.com/waardenlijsten/1",
+            naam="first item",
+            naam_meervoud="first items",
+            definitie="This is some information about the first item.",
+            order=0,
+            oorsprong=InformationCategoryOrigins.value_list,
+        )
+        self.information_category_custom_entry = InformationCategoryFactory.create(
+            identifier="https://www.example.com/waardenlijsten/2",
+            naam="second item",
+            naam_meervoud="second items",
+            definitie="This is some information about the second item.",
+            order=1,
+            oorsprong=InformationCategoryOrigins.custom_entry,
+        )
+
+    def test_list_informatie_categorie(self):
+        list_url = reverse("api:informationcategory-list")
+        response = self.client.get(list_url)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        data = response.json()
+        self.assertEqual(data["count"], 2)
```

Comment:
```suggestion
        list_url = reverse("api:informationcategory-list")

        response = self.client.get(list_url)

        self.assertEqual(response.status_code, status.HTTP_200_OK)
        data = response.json()
        self.assertEqual(data["count"], 2)
```
the visual separation between

```
setup
setup

act

assert
assert
```
is important to have well-readable tests :)